lncfg/protocol.go

```diff
@@ -17,10 +17,20 @@ type ProtocolOptions struct {
 	// (channels larger than 0.16 BTC) channels, which is the opposite of
 	// mini.
 	WumboChans bool `long:"wumbo-channels" description:"if set, then lnd will create and accept requests for channels larger chan 0.16 BTC"`
+
+	// NoAnchors should be set if we don't want to support opening or accepting
+	// channels having the anchor commitment type.
+	NoAnchors bool `long:"no-anchors" description:"disable support for anchor commitments"`
```

Comment:
not sure how specific we want to be here, but does it make sense to clarify that this is `anchor_zero_fee_htlc` vs just `anchor_outputs`?

Reply:
I think users interacting with this flag don't have to know about the difference.